share/completions/zabbix_proxy.fish

```diff
@@ -0,0 +1,70 @@
+set -l runtime config_cache_reload \
+    snmp_cache_reload \
+    housekeeper_execute \
+    diaginfo \
+    diaginfo= \
+    log_level_increase \
+    log_level_increase= \
+    log_level_decrease \
+    log_level_decrease=
+
+
+function __fish_has_target
+    if string match -rq '=' (commandline)
+        return 0
+    end
+    return 1
+end
+
+function __fish_string_in_command -a ch
+    if string match -rq $ch (commandline )
```

Comment:
You don't need an `if` statement here, just do it:

    function __fish_string_in_command -a ch
          string match -rq $ch (commandline)
    end

Also I suggest using globs instead of a regex, since it's not obvious to the caller that regex characters will be respected:

    function __fish_string_in_command -a ch
          string match -q "*$ch*" (commandline)
    end